engine/lib/localStorageRepository.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+var logger = require("./logger")
+const fileUtil = require("./fileUtil")

```

Comment:
If we're using `const` for one import, we should probably use it for them all.
```suggestion
const logger = require("./logger")
const fileUtil = require("./fileUtil")
```

Reply:
Good catch! updated to use const